src/services/boardService.js

```diff
@@ -0,0 +1,12 @@
+
+function makeBoard(ranks, files) {
+  return ranks.reduce((board, rank) => {
+    files.forEach(file => {
+      const coordinate = `${file}${rank}`;
+      board.push({ coordinate: coordinate, piece: null});
+    });
+    return board;
+  }, []);
+}
+
+module.exports = (makeBoard);
```

Comment:
Think about future development, as you will be adding more functions to this service. It can be changed down the road if needed, just always be thinking about that - not to much though :-)